shunit2

```diff
@@ -630,7 +630,12 @@ failNotEquals() {
   shunit_actual_=$2
 
   shunit_message_=${shunit_message_%% }
-  _shunit_assertFail "${shunit_message_:+${shunit_message_} }expected:<${shunit_expected_}> but was:<${shunit_actual_}>"
+  if command [ $(echo "${shunit_expected_}" | wc -l) -gt 10 ] || command [ $(echo "${shunit_actual_}" | wc -l) -gt 10 ]; then
+      diff_output_=$(diff <(echo "${shunit_expected_}") <(echo "${shunit_actual_}"))
```

Comment:
s/diff_output_/shunit_diff_/ -- to meet the style guide for "public function, local variable".
https://github.com/kward/shunit2/wiki/Coding-standards#variable-and-function-names

Reply:
Done